src/config-site/steppenwolf.cmake

```diff
@@ -14,3 +16,25 @@ VISIT_OPTION_DEFAULT(VISIT_NO_ALLBUILD ON TYPE BOOL)
 
 SET(CMAKE_SUPPRESS_REGENERATION TRUE)
 SET(CMAKE_SKIP_INSTALL_ALL_DEPENDENCY TRUE)
+
+if(USE_REDUCED_BUILD) # this is just a var I create when I need it
```

Comment:
Would this catch-all var be useful to others in general? If so, maybe define it at top and migrate this logic there?

Reply:
Perhaps, but I think use case may be different for each developer, especially in terms of the SELECTED plugins being built, and I've added windows-specific option in my if-block.